### PR TITLE
github: workflow: Add build and test python binding

### PR DESCRIPTION
**Pull request by moonlight83340**
Add build and test python binding.
This commit aim to help to found if there are any build error in python binding.

### CODE REVIEW

File: .github/workflows/build-test-python.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Build libcsp examples
+        run: python3 examples/buildall.py ${{ matrix.arch }} --build-system=${{ matrix.buildsystem }}
+
+      - name: Build libcsp with python binding with cmake
+        if: ${{ matrix.buildsystem == 'cmake' }}
+        run: |
+          cmake -GNinja -B buildBinding -DCSP_ENABLE_PYTHON3_BINDINGS=1 -DCSP_USE_RTABLE=1 -DCMAKE_POSITION_INDEPENDENT_CODE=ON && ninja -C buildBinding
```
Comment (yashi) [inline .github/workflows/build-test-python.yml]: Do we need `DCMAKE_POSITION_INDEPENDENT_CODE=`?

Also, can we just use `builddir` for `-B`?

**Review by yashi — CHANGES_REQUESTED** [review]

File: .github/workflows/build-test-python.yml (hunk 1 above)
Comment (moonlight83340) [reply .github/workflows/build-test-python.yml]: No we don't need that option, yes we can use `builddir `👍

**Review by yashi — APPROVED** [review]

File: .github/workflows/build-test-python.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+      - name: Build libcsp with python binding with meson
+        if: ${{ matrix.buildsystem == 'meson' }}
+        run: |
+          meson setup builddir -Denable_python3_bindings=true -Duse_rtable=true && ninja -C builddir
+
+      - name: Run ZMQ Python binding Test
+        run: |
+          build/examples/zmqproxy &
+          LD_LIBRARY_PATH=builddir PYTHONPATH=builddir python3 examples/python_bindings_example_server.py &
```
Comment (yashi) [inline .github/workflows/build-test-python.yml]: Do we need `LD_LIBRARY_PATH`?

**Review by yashi — CHANGES_REQUESTED** [review]

File: .github/workflows/build-test-python.yml (hunk 2 above)
Comment (moonlight83340) [reply .github/workflows/build-test-python.yml]: No indeed. Removed !

**Review by yashi — APPROVED** [review]

**Pull request merged**